CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.11.1-0.30b1](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v1.11.1-0.30b1) - 2022-04-21
 
 
-
+- Implement MetricReader temporality controls
```

Comment:
if possible here i would capture the change in method signatures to give readers more information about what's changing

Reply:
Fixed